src/pps/pps.proto

```diff
@@ -513,21 +497,21 @@ message DeleteJobRequest {
 
 message StopJobRequest {
   Job job = 1;
-  pfs_v2.Commit output_commit = 2;
   string reason = 3;
 }
 
 message UpdateJobStateRequest {
```

Comment:
Yeah, I think there is an opportunity to decrease the number of fields in a lot of these protos by refactoring out some of the common fields in the Stored* protos.